tests/rt.sh

```diff
@@ -752,6 +752,13 @@ elif [[ $MACHINE_ID = hercules ]]; then
 
 elif [[ $MACHINE_ID = jet ]]; then
 
+  echo "=======Running on $(lsb_release -is)======="
+CurJetOS="$(lsb_release -is)"
```

Comment:
I take that back. You'll want something more like:
```
  CurJetOS=$(lsb_release -is)
  if [[ ${CurJetOS} == "CentOS" ]]; then
```

Reply:
Ok, done. @BrianCurtis-NOAA